test/validation/api/crypto/odp_crypto_test_inp.c

```diff
@@ -1566,6 +1653,9 @@ static void test_auth_hash_in_auth_range(odp_auth_alg_t auth,
 	if (create_hash_test_reference(auth, capa, &ref, digest_offset, 0))
 		return;
 
+	crypto_range.offset = 0;
+	crypto_range.length = ref.length;
```

Comment:
The length should be converted to bits if the cipher supports bit mode.

Reply:
Actually, there is no cipher used here, only authentication. Auth range should be in bits if the auth algorithm uses bit mode, which is a bug and fixed in v2. In v2 I also reinitialize auth range after the second ref creation. It is not needed now but makes things more robust in case something changes in the ref creation.